homeassistant/components/mqtt/strings.json

```diff
@@ -1,4 +1,10 @@
 {
+  "issues": {
+    "deprecated_yaml": {
+      "title": "Your manually configured MQTT {platform}(s) needs attention",
+      "description": "Manually configured MQTT {platform}(s) found under platform key `{platform}`.\n\nPlease move the configuration to the mqtt integration key, see {more_info_url}.."
```

Comment:
```suggestion
      "description": "Manually configured MQTT {platform}(s) found under platform key `{platform}`.\n\nPlease move the configuration to the mqtt integration key, see {more_info_url}."
```

Reply:
Does the user need to restart Home Assistant or reload the YAML after moving the config? We normally include that instruction in the description.

Reply:
> Does the user need to restart Home Assistant or reload the YAML after moving the config? We normally include that instruction in the description.

I'll have a look at that.

Reply:
User needs to restart Home Assistant, Added this.